isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
+         spacecraft position, like the overexisting flag estimates the camera pointing with a polynomial. As such, this option saves memory and time at the expense of 
+         some loss in accuracy and this loss can be mitigated by switching it off for the final run.
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+         jigsaw fromlist=cubes_for_ground_update.lis cnet=grounded_relative_reg.net onet=grounded_rr_jig1.net update=no file_prefix=jig1rr sigma0=1.0e-10 maxits=10 \
+                twist=yes camsolve=accelerations overexisting=yes camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01 \
+                radius=yes point_radius_sigma=500 spsolve=positions overhermite=yes spacecraft_position_sigma=1000 
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>a final ground control</b> network covering Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         This last example is of a final jigsaw run of a grounded network. In this example all network adjustments are done, the bundle is converging, the resulting 
+         residuals are acceptable, and therefore we are ready to update the camera pointing on the cubes. In the final run you can turn off overexisting and overhermite, 
+         if you want to regain the accuracy, but note that the bundle solution will differ from your previous run. Therefore, it is advisable to conduct one intermediate 
+         run to check the bundle is acceptable without these flags before updating the cubes. Finally, during the final run we turn on the error propagation flag. This 
+         provides the variance-covariance matrix of the parameters, from which uncertainties can be computed. This is valuable if you plan to compute certainties for 
+         your update cubes camera pointing (or any kernels resulting from these updated camera pointings).
+        </p>
+        <p>
+         If you want to double check the update was completed, see cathist or catlab (search for ‘Jigged’).
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+         jigsaw fromlist=cubes_for_ground_update.lis cnet=grounded_relative_reg.net onet=grounded_rr_jig1_ErrProp.net update=yes file_prefix=jig1rrEP sigma0=1.0e-10 maxits=10 \
+                twist=yes camsolve=accelerations camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01 \
+                radius=yes point_radius_sigma=500 spsolve=positions spacecraft_position_sigma=1000 errorpropagation=yes 
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+
```

Comment:
I realize I have added no supplementary files or GUI images. Please let me know if you think adding those would enhance the examples.

Reply:
I don't think we need GUI images, but it would be nice if you could provide the bundleout.txt for each solution as it contains the human-readable information that would help contextualize the data.

Reply:
Can I include the entire bundleout.txt file? Each one is about ~25M, so that would be almost a Gig for all three examples. 

Alternatively I could shorten the files, just keep the full headers and a few example lines from each section?